### ISSUE TITLE
Rework AB dates to allow for multiple specials

### ISSUE THREAD

**Issue by katherinepdavis** [BlueprintKansas/ksvotes.org#429]
And all to be added/removed/changed via translations file

**Comment by karpet** [comment]
yes I was thinking this morning about the need for this. I think what we want is to be able to define an array (list) of elections with dates. It might require a combination of translations and some other mechanism. Maybe even a separate `elections.json` file.

**Comment by katherinepdavis** [comment]
I did some research into how many specials there are in a given year, and the answer was "quite a few." Apparently there are lots of school bond issues and that kind of thing in the spring now that they've changed local elections to the fall. 

A better solution here may be to allow user input on the date on the form, which is more like replicating a paper form anyway. Is that workable? Any sense of how many hours it would take @frankwiles ?

**Comment by katherinepdavis** [comment]
Yeah some data validation is probably required. The only input we would
want would be a date.

**Comment by katherinepdavis** [comment]
Bumping this one - I got another request for it today. How tough would it be to add a "Special" option and then let the user enter a date?

**Comment by karpet** [comment]
We can tackle this as part of v3
